examples/basic.rs

```diff
@@ -3,7 +3,7 @@ extern crate hashicorp_vault as vault;
 fn main() {
     let host = "http://localhost:8200";
     let token = "test12345";
-    let client = vault::Client::new(host, token).unwrap();
+    let client = vault::Client::new(host, token, &[""]).unwrap();
```

Comment:
I'm also in favor of using the builder pattern for anything more than two args here.